tango_simlib/model.py

```diff
@@ -529,6 +533,45 @@ def action_handler(model, data_input=None, tango_dev=None):
         return action_handler
 
 
+class PopulateModelProperties(object):
+    """Used to populate/update model properties
+
+    Populates the model properties using the data from the TANGO device information
+    captured in the POGO generated xmi file.
+
+    Attributes
+    ----------
+    parser_instance : Parser instance
+        The Parser object which reads an xmi/xml/json file and parses it into device
+        attributes, commands, and properties.
+    sim_model :  Model instance
+        An instance of the Model class which is used for simulation of simple attributes.
+
+    """
+    def __init__(self, parser_instance, tango_device_name, sim_model=None):
+        self.parser_instance = parser_instance
+        if sim_model:
+            if isinstance(sim_model, Model):
+                self.sim_model = sim_model
+            else:
+                raise SimModelException("The sim_model object passed is not an "
+                    "instance of the class mkat_tango.simlib.model.Model")
```

Comment:
Please fix the indentation here. This should line up with the start of the string on the previous line.